docs/api/doc.md

```diff
@@ -469,4 +477,4 @@ doc.on('error', function(error) { ... })
 
 `error` -- [ShareDBError]({{ site.baseurl }}{% link api/sharedb-error.md %})
 
-> The error that occurred
\ No newline at end of file
+> The error that occurred
```

Comment:
IDK why this is part of the diff. FWIW I used the GitHub online editor, maybe it added something invisible.